[CI] CsvIT test {csv-spec:inlinestats.doubleShadowingWithEval} failing

**Build Scans:**
- [elasticsearch-intake #42940 / part3](https://gradle-enterprise.elastic.co/s/k3zf2xigovyyc)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:esql:internalClusterTest" --tests "org.elasticsearch.xpack.esql.CsvIT" -Dtests.method="test {csv-spec:inlinestats.doubleShadowingWithEval}" -Dtests.seed=14CB4B386A1F5494 -Dtests.locale=mg-MG -Dtests.timezone=Asia/Singapore -Druntime.java=26
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.xpack.esql.CsvIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('test%20%7Bcsv-spec%3Ainlinestats.doubleShadowingWithEval%7D'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
org.elasticsearch.ElasticsearchTimeoutException: java.util.concurrent.TimeoutException: Timeout waiting for task.
```

**Issue Reasons:**
- [main] 1 consecutive failures in test test {csv-spec:inlinestats.doubleShadowingWithEval}
- [main] 1 consecutive failures in step part3
- [main] 1 consecutive failures in pipeline elasticsearch-intake

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

This should be fixed by https://github.com/elastic/elasticsearch/pull/153171

Found these in the [log](https://gradle-enterprise.elastic.co/s/k3zf2xigovyyc/tests/task/:x-pack:plugin:esql:internalClusterTest/details/org.elasticsearch.xpack.esql.CsvIT/test%20%7Bcsv-spec:inlinestats.doubleShadowingWithEval%7D/1/output)
```
[2026-07-07T11:57:27,439][INFO ][o.e.x.e.CsvIT            ][test] [csv-spec:inlinestats.doubleShadowingWithEval] before test
Jol 07, 2026 7:57:27 PM com.carrotsearch.randomizedtesting.RandomizedRunner$QueueUncaughtExceptionsHandler uncaughtException
WARNING: Uncaught exception in thread: Thread[#98,elasticsearch[node_0][esql_worker][T#4],5,TGRP-CsvIT]
java.lang.AssertionError: java.lang.IllegalStateException: can't release already released object [BytesRefArrayVector[positions=2]]
	at __randomizedtesting.SeedInfo.seed([14CB4B386A1F5494]:0)
	at org.elasticsearch.core.Releasables.closeExpectNoException(Releasables.java:64)
	at org.elasticsearch.compute.data.Page.releaseBlocks(Page.java:299)
	at org.elasticsearch.compute.operator.topn.TopNOperator.addInput(TopNOperator.java:416)
	at org.elasticsearch.compute.operator.topn.ParallelTopNOperator.processPagesWithCurrentThread(ParallelTopNOperator.java:215)
	at org.elasticsearch.compute.operator.topn.ParallelTopNOperator.finish(ParallelTopNOperator.java:243)
	at org.elasticsearch.compute.operator.Driver.closeEarlyFinishedOperators(Driver.java:420)
	at org.elasticsearch.compute.operator.Driver.runSingleLoopIteration(Driver.java:361)
	at org.elasticsearch.compute.operator.Driver.run(Driver.java:204)
	at org.elasticsearch.compute.operator.Driver$1.doRun(Driver.java:546)
	at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:27)
	at org.elasticsearch.compute.operator.DriverScheduler$1.doRun(DriverScheduler.java:57)
	at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:27)
	at org.elasticsearch.common.util.concurrent.TimedRunnable.doRun(TimedRunnable.java:35)
	at org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:1114)
	at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:27)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1090)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:614)
	at java.base/java.lang.Thread.run(Thread.java:1516)
Caused by: java.lang.IllegalStateException: can't release already released object [BytesRefArrayVector[positions=2]]
	at org.elasticsearch.compute.data.AbstractNonThreadSafeRefCounted.decRef(AbstractNonThreadSafeRefCounted.java:60)
	at org.elasticsearch.compute.data.AbstractNonThreadSafeRefCounted.close(AbstractNonThreadSafeRefCounted.java:80)
	at org.elasticsearch.core.Releasables.close(Releasables.java:39)
	at org.elasticsearch.core.Releasables.close(Releasables.java:48)
	at org.elasticsearch.compute.data.OrdinalBytesRefBlock.closeInternal(OrdinalBytesRefBlock.java:191)
	at org.elasticsearch.compute.data.AbstractNonThreadSafeRefCounted.decRef(AbstractNonThreadSafeRefCounted.java:66)
	at org.elasticsearch.compute.data.AbstractNonThreadSafeRefCounted.close(AbstractNonThreadSafeRefCounted.java:80)
	at org.elasticsearch.core.Releasables.close(Releasables.java:39)
	at org.elasticsearch.core.Releasables.close(Releasables.java:48)
	at org.elasticsearch.core.Releasables.closeExpectNoException(Releasables.java:62)
	... 17 more

```

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -646,9 +646,6 @@ tests:
 - class: org.elasticsearch.xpack.stateless.reshard.StatelessReshardIT
   method: testMultiTermVectorsApiRealtimeGet
   issue: https://github.com/elastic/elasticsearch/issues/150101
-- class: org.elasticsearch.xpack.esql.CsvIT
-  method: test {csv-spec:inlinestats.doubleShadowingWithEval}
-  issue: https://github.com/elastic/elasticsearch/issues/153096
 - class: org.elasticsearch.search.telemetry.SearchCoordinatorPhaseShardBytesMetricsIT
   method: testBatchedQueryPhaseShardErrorRecordsBothRequestAndResultBytes
   issue: https://github.com/elastic/elasticsearch/issues/153109

